core/src/main/java/eu/tsystems/mms/tic/testframework/report/utils/ExecutionContextController.java

```diff
@@ -146,10 +162,20 @@ public static void setCurrentSessionContext(final SessionContext sessionContext)
         CURRENT_SESSION_CONTEXT.set(sessionContext);
     }
 
-    public static Optional<SessionContext> getCurrentSessionContext() {
+    /**
+     * Returns the current active session context, set by WebdriverProxy
+     */
+    public static Optional<SessionContext> getSessionContextForThread() {
         return Optional.ofNullable(CURRENT_SESSION_CONTEXT.get());
     }
 
+    /**
+     * @deprecated Use {@link #getSessionContextForThread()} instead
+     */
+    public static SessionContext getCurrentSessionContext() {
+        return getSessionContextForThread().orElse(null);
+    }
+
     public static void clearCurrentSessionContext() {
```

Comment:
You changed the naming of the getters. Whats about this method?

Reply:
I changed the signature of this method before (in another branch) but this makes connectors like `selenoid-connector` incompatible. Thats why I changed it back to its original behaviour but made it `@deprecated`

Reply:
Yes, but i mean the methdod `clearCurrentSessionContext`. You added new getters with `...ForThread`. Should this method not also be changed?